Watchman.Integrations/Logging/SerilogInitializer.cs

```diff
@@ -24,9 +20,9 @@ public static ILogger Initialize(IMongoDatabase mongoDatabase)
                 .Enrich.WithThreadId()
                 .Enrich.WithMachineName()
                 .Enrich.WithEnvironmentUserName()
-                .WriteTo.RollingFile(new JsonFormatter(closingDelimiter: ","), "logs/all/log-{Date}.json", shared: true, flushToDiskInterval: TimeSpan.FromSeconds(15), restrictedToMinimumLevel: LogEventLevel.Verbose)
-                .WriteTo.RollingFile(new JsonFormatter(closingDelimiter: ","), "logs/warningplus/log-{Date}.json", shared: true, flushToDiskInterval: TimeSpan.FromSeconds(15), restrictedToMinimumLevel: LogEventLevel.Warning)
-                .WriteTo.RollingFile(new JsonFormatter(closingDelimiter: ","), "logs/errorplus/log-{Date}.json", shared: true, flushToDiskInterval: TimeSpan.FromSeconds(15), restrictedToMinimumLevel: LogEventLevel.Error)
+                .WriteTo.File(new JsonFormatter(closingDelimiter: ","), "logs/all/log-{Date}.json", rollingInterval: RollingInterval.Day, shared: true, flushToDiskInterval: TimeSpan.FromSeconds(15), restrictedToMinimumLevel: LogEventLevel.Verbose)
```

Comment:
nie pamiętam czy to jest w tej konfiguracji czy już nie 
ale dorzuć max rozmiar logów 
powiedzmy że 10 albo 20GB dla całości 
(jak dojdzie do limitu to żeby stare sie usuwały kiedy nowe będą przychodzić, ten mechanizm sie jakoś nazywał ale zapomniałem nazwy, na 90% to jest do ogarnięcia w konfiguracji tutaj)

Reply:
wydaje mi się, że da się jedynie ustawić limit dla pojedynczego pliku (obecnie jest odmyślna wartość = 1GB)
nie widzę za bardzo opcji po rozmiarze całości